concepticondata/conceptlists/Capell-1956-558.tsv

```diff
@@ -0,0 +1,559 @@
+Capell-1956-558-70			70	bullroarer	Capell1956[18]
+Capell-1956-558-71	2102	BURN	71	burn	Capell1956[18]
+Capell-1956-558-72			72	burrawang (cycas) fruit	Capell1956[18]
+Capell-1956-558-73	2949	LAIR	73	burrow	Capell1956[19]
+Capell-1956-558-74	2811	BUSHFIRE	74	bush-fire	Capell1956[19]
+Capell-1956-558-75	1791	BUTTERFLY	75	butterfly	Capell1956[19]
+Capell-1956-558-76	94	GRADUALLY	76	by and by	Capell1956[19]
+Capell-1956-558-77	663	BAY	77	bay	Capell1956[20]
+Capell-1956-558-78	1531	BRANCH	78	branch	Capell1956[20]
+Capell-1956-558-79	1084	CALL	79	call	Capell1956[20]
+Capell-1956-558-80	1105	CAMP	80	camp	Capell1956[20]
+Capell-1956-558-81	1051	CAN	81	can	Capell1956[21]
+Capell-1956-558-82	1970	CANOE	82	canoe	Capell1956[21]
+Capell-1956-558-83	1662	CAPE	83	cape	Capell1956[21]
+Capell-1956-558-84			84	capsize	Capell1956[21]
+Capell-1956-558-85	700	CARRY	85	carry	Capell1956[22]
+Capell-1956-558-86	702	CATCH	86	catch	Capell1956[22]
+Capell-1956-558-87	2247	CATERPILLAR	87	caterpillar	Capell1956[22]
+Capell-1956-558-88	3777	CATFISH	88	catfish	Capell1956[22]
+Capell-1956-558-89	1155	CAVE	89	cave	Capell1956[23]
+Capell-1956-558-90	1974	CENTIPEDE	90	centipede	Capell1956[23]
+Capell-1956-558-91	1188	CHARCOAL	91	charcoal	Capell1956[23]
+Capell-1956-558-92	1730	CHEEK	92	cheek	Capell1956[23]
+Capell-1956-558-93	1592	CHEST	93	chest	Capell1956[24]
+Capell-1956-558-94	1510	CHIN	94	chin	Capell1956[24]
+Capell-1956-558-95			95	chiton	Capell1956[24]
+Capell-1956-558-96	497	SCAR	96	cicatrix	Capell1956[24]
+Capell-1956-558-97	393	CIRCUMCISION	97	circumcision	Capell1956[25]
+Capell-1956-558-98	1399	CLAY	98	clay	Capell1956[25]
+Capell-1956-558-99	704	CLEAN	99	clean	Capell1956[25]
+Capell-1956-558-100	1310	CLEVER	100	clever	Capell1956[25]
+Capell-1956-558-101	1102	CLIMB	101	climb	Capell1956[26]
+Capell-1956-558-102	1046	SHUT	102	close (adj.)	Capell1956[26]
+Capell-1956-558-103	1489	CLOUD	103	cloud	Capell1956[26]
+Capell-1956-558-104	1763	CLUB	104	club	Capell1956[26]
+Capell-1956-558-105	3013	BLACK COCKATOO	105	cockatoo (black)	Capell1956[27]
+Capell-1956-558-106	3009	WHITE COCKATOO	106	cockatoo (white)	Capell1956[27]
+Capell-1956-558-107	970	COCONUT	107	coconut	Capell1956[27]
+Capell-1956-558-108	2011	COFFIN	108	coffin	Capell1956[27]
+Capell-1956-558-109	1287	COLD	109	cold	Capell1956[28]
+Capell-1956-558-110			110	cold wind	Capell1956[28]
+Capell-1956-558-111	1446	COME	111	come	Capell1956[28]
+Capell-1956-558-112			112	consider	Capell1956[28]
+Capell-1956-558-113	3179	CORAL ROCK	113	coral	Capell1956[29]
+Capell-1956-558-114			114	corroboree	Capell1956[29]
+Capell-1956-558-115	761	COVER	115	cover	Capell1956[29]
+Capell-1956-558-116	2348	CRAB	116	crab	Capell1956[29]
+Capell-1956-558-117	467	CRAYFISH	117	crayfish	Capell1956[30]
+Capell-1956-558-118	990	CREEK (TIDAL)	118	creek	Capell1956[30]
+Capell-1956-558-119	1857	CROCODILE	119	crocodile	Capell1956[30]
+Capell-1956-558-120	297	CROOKED	120	crooked	Capell1956[30]
+Capell-1956-558-121	1367	CROSS	121	cross (v.)	Capell1956[31]
```

Comment:
This should be TRAVERSE. I've opened an issue about this (#1316 ) as this has been mapped wrongly in some other cases as well.